routing/index_graph_starter.cpp

```diff
@@ -19,162 +48,373 @@ Junction CalcProjectionToSegment(Segment const & segment, Junction const & junct
 
 namespace routing
 {
-// static
-Segment constexpr IndexGraphStarter::kStartFakeSegment;
-Segment constexpr IndexGraphStarter::kFinishFakeSegment;
+void IndexGraphStarter::AddFakeVertex(Segment const & existentSegment, Segment const & newSegment,
+                                      FakeVertex const & newVertex, bool isOutgoing,
+                                      bool isPartOfReal, Segment const & realSegment)
+{
+  auto const & segmentFrom = isOutgoing ? existentSegment : newSegment;
+  auto const & segmentTo = isOutgoing ? newSegment : existentSegment;
+  m_fake.m_outgoing[segmentFrom].insert(segmentTo);
```

Comment:
This method works only with m_fake, how about to transform it to a method of FakeGraph?

Reply:
done